mentoring-app/src/app/header/header.component.html

```diff
@@ -1 +1,9 @@
-<p>header works!</p>
+<mat-toolbar class="header">
+    <div class="header_logo">
+        <img src="../../assets/images/courses-logo.png" alt="logo">
```

Comment:
Can be simplified to <img src="assets/images/courses-logo.png" alt="logo">, thanks to default configuration of angular cli (in angular.json) that moves all of our assets into the dist folder.

Reply:
fixed